.github/workflows/asset_dispatch.yml

```diff
@@ -0,0 +1,38 @@
+# Triggered on dispatching the action
+# Automated releases handle asset creation in automated_release.yml
+name: Asset - on dispatch; build and publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      image_tag:
+        description: Asset Name
+        required: true
+        default: x.y.z-uat.1
+
+jobs:
+  build_and_release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Install dependencies
+        run: >-
+          yarn install
+      - name: Compile build
+        run: >-
+          ./compile-build.sh release.tar.gz dist
+      - name: Upload release.gz
+        id: upload-release-gz
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }} # Pull the URL from the release step
```

Comment:
Might be me mis-reading this but not sure where this upload_url is being pulled from? Since there isnt a release event in this workflow.

Reply:
Ah I see the issue, if this is copied partially from the manual_build file, that workflow runs on release creation so has a release event, for this one you will need to add a manual release creation step.

Reply:
Yeah, have refactored this using an action that is actively developed - the GitHub created one is archived.